optimum/exporters/openvino/model_configs.py

```diff
@@ -954,3 +955,21 @@ def patch_model_for_export(
         self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
     ) -> "ModelPatcher":
         return GptNeoxModelPatcher(self, model, model_kwargs=model_kwargs)
+
+
+@register_in_tasks_manager(
+    "gemma2",
+    *[
+        "feature-extraction",
+        "feature-extraction-with-past",
+        "text-generation",
+        "text-generation-with-past",
+        "text-classification",
+    ],
+    library_name="transformers",
+)
```

Comment:
also can you add the `MIN_TRANSFORMERS_VERSION`

Reply:
why? we do not added it before. why it is the moment to start doing that?

Reply:
I though it is used only for case, when model support in transformers added several releases ago, but optimum support only specific model version beginning from different version (e.g. old model is nontrecable), but for this model we has aligned addition model version and supported